ember/ember-cli-build.js

```diff
@@ -27,5 +27,17 @@ module.exports = function (defaults) {
         package: 'qunit',
       },
     ],
+    packagerOptions: {
+      webpackConfig: {
+        plugins: [
+          new BrotliPlugin({
+            asset: '[path].br[query]',
+            test: /\.(js|css|html|svg)$/,
+            threshold: 10240,
+            minRatio: 0.8,
+          }),
+        ],
+      },
+    },
```

Comment:
I love how easy this is!

Does this compress in development too? If so, please make sure it only compresses in production environments..